docs/install/install_everest_helm_charts.md

```diff
@@ -0,0 +1,178 @@
+# Install Percona Everest using Helm
+
+This section explains how to install Percona Everest using [Helm](https://helm.sh/) as an alternative to `everestctl`. Helm charts simplify the deployment process by packaging all necessary resources and configurations, making them ideal for automating and managing installations in Kubernetes environments.
+
+Percona Helm charts can be found in [percona/percona-helm-charts](https://github.com/percona/percona-helm-charts) repository in Github.
```

Comment:
Shouldn't we link to the Percona Everest Helm chart instead?

Reply:
ok

Reply:
Fixed